### PR TITLE
`<FormField/>`- stylable migration 

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
stylable migration 

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [x] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [x] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5648/

File: src/FormField/test/FormField.spec.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {
   cleanup,
 } from '../../../test/utils/react';
 
-import styles from '../FormField.scss';
+import styles from '../FormField.st.css';
```
Comment (TalZa1810) [inline src/FormField/test/FormField.spec.js:10]: This is used only in line 145. 
Should we convert this test to a visual one?
It feels a bit redundant, I thought maybe deleting it completly.
WDYT? 

```
      it('should apply minHeight on label wrapper when there is no children', async () => {
        const { driver } = render(<FormField label="Text" />);
        expect(await driver.exists()).toEqual(true);
        expect(
          (await driver.element()).querySelector(`.${styles.minLabelHeight}`),
        ).not.toBeNull();
      });
```

Comment (EgoziE) [reply src/FormField/test/FormField.spec.js:10]: delete

**Comment by TalZa1810** [comment]
I know that I don't use the stylable states. I want to do it in a different PR since it's a big change.

**Pull request merged**